pkgs/development/ocaml-modules/cmdliner/1_1.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl, ocaml, findlib, ocamlbuild, topkg, result }:
+
+assert (lib.versionAtLeast ocaml.version "4.08");
```

Comment:
you can use minimalOCamlVersion instead for a better error message

Reply:
Done, thanks.